src/Microsoft.DotNet.ProjectModel/ProjectReader.cs

```diff
@@ -246,6 +243,38 @@ public Project ReadProject(Stream stream, string projectName, string projectPath
             return project;
         }
 
+        internal static string NormalizeProjectDirectoryPath(string path)
+        {
+            if (File.Exists(path) &&
```

Comment:
This code was moved from WorkspaceContext to here with 1 line changing in it.

Normally at the end of a milestone you don't make unnecessary refactorings.  You usually have other, more important, things to attend to.  And you want to minimize churn in the code base, limit bugs.